content/en/docs/ops/deployment/multicluster-considerations/index.md

```diff
@@ -0,0 +1,155 @@
+---
+title: Multicluster Deployment Considerations
+description: Describes considerations when configuring your Multicluster Istio deployment.
+weight: 20
+keywords:
+  - multiple-clusters
+  - control-plane
+  - networks
+owner: istio/wg-environments-maintainers
+test: n/a
+---
+
+Before you install Istio on {{< gloss "multicluster" >}}multiple clusters{{< /gloss >}}, there are some things to
+consider regarding the requirements of your system which may impact your installation choices.
+
+## Control Planes
+
+This section describes things you’ll need to consider when deploying an Istio {{< gloss >}}control plane{{< /gloss >}}
+in your environment.
+
+### Configuration
+
+An Istio control plane in a {{< gloss >}}primary cluster{{< /gloss >}} receives its configuration
+(i.e. `Service`/`ServiceEntry`, `DestinationRule`, etc.) from the Kubernetes API Server residing in the same
+{{< gloss >}}cluster{{< /gloss >}}.
+
+When a mesh contains multiple primary clusters, each control plane has an independent source of configuration.
+
+{{< image width="80%"
```

Comment:
Should this image clarify that we're reading _istio_ configuration from the local api server? Could be confusing later when we introduce reading k8s objects from remote api servers.

Reply:
Actually, it's plenty clear in that diagram.